llama-index-integrations/vector_stores/llama-index-vector-stores-redis/llama_index/vector_stores/redis/base.py

```diff
@@ -266,7 +266,7 @@ def add(self, nodes: List[BaseNode], **add_kwargs: Any) -> List[str]:
             key.strip(self._index.prefix + self._index.key_separator) for key in keys
         ]
 
-    def delete(self, ref_doc_id: str, **delete_kwargs: Any) -> None:
```

Comment:
Delete is method on the base class, we can't remove it

Reply:
i see ,  thx